src/doc/dev_manual/pr_create.rst

```diff
@@ -0,0 +1,106 @@
+Creating a Pull Request
```

Comment:
So, I wasn't paying too much attention when vieweing these changes on GitHub via web browser. The little plus signs down the left column indicat beginning of lines. When I checked out this branch to edit locally, it was more obvious to me. 

Our policy is to keep lines in `.rst` files to 80 columns or there-abouts. I checked to make sure an indeed our [Contributing](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/gui_manual/Contributing/index.html) does indeed mention that burried in many bullets listed there.

Reply:
Neglected to mention...this actually also has a benefit to the git's diffing algorithm when storing file history as well as presenting diffs to developers with diff tools.

Reply:
I updated the files with explicit line breaks around 80 columns. Except for lists, which require the entire list item to be on one line.